core/win32/os.c

```diff
@@ -6979,8 +6979,8 @@ os_seek(file_t f, int64 offset, int origin)
     case OS_SEEK_END:
         {
             uint64 file_size = 0;
-            bool res = os_get_file_size_by_handle(f, &file_size);
```

Comment:
I think something like `size_res` would be more descriptive.

Reply:
changed